src/components/Atoms/Input/Input.js

```diff
@@ -2,45 +2,61 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import styled, { css } from 'styled-components';
 
+import alertIcon from './assets/error-alert-icon-red.svg';
 import Label from '../Label/Label';
 import ErrorText from '../ErrorText/ErrorText';
-import spacing from '../../../theme/shared/spacing';
 import zIndex from '../../../theme/shared/zIndex';
 
 // This seems to get a decent approximation of the necessary width (without resorting to measuring
 //  the element with JS)
-const getPrefixWidth = prefixLength => `calc(${spacing('m')} + (${prefixLength} * ${spacing('sm')}))`;
+const getPrefixWidth = prefixLength => `calc(1.5rem + (${prefixLength} * 0.5rem))`;
 
 const InputField = styled.input`${({ theme, error, prefixLength }) => css`
   position: relative;
   box-sizing: border-box;
   width: 100%;
   height: 48px;
-  padding: ${spacing('md')} ${spacing('m')};
+  padding: 1rem 2.4rem 1rem 1.5rem;
   ${prefixLength > 0 ? `padding-left: ${getPrefixWidth(prefixLength)};` : ''}
   background-color: ${theme.color('grey_light')};
   border: 1px solid;
   border-color: ${error ? theme.color('red') : theme.color('grey_medium')};
   box-shadow: none;
   appearance: none;
   color: ${theme.color('black')};
-  border-radius: ${spacing('sm')};
+  border-radius: 0.5rem;
   font-size: inherit;
   z-index: 2;
   font-family: ${theme.fontFamilies(theme.font.regular)};
 
   :focus {
     border: 1px solid ${theme.color('grey_for_forms')};
   }
-
-  @media ${theme.allBreakpoints('M')} {
-    max-width: 290px;
-  }
 `}`;
 
 const InputWrapper = styled.div`
   position: relative;
   font-size: ${({ theme }) => theme.fontSize('m')};
+
+  ${({ error }) => error && css`
+    &::after {
+      content: '';
+      position: absolute;
+      top: 50%;
+      right: 0.5rem;
+      transform: translateY(-50%);
+      background: url(${alertIcon}) center/contain no-repeat;
+      --iconSize: 19px;
+      width: var(--iconSize);
+      height: var(--iconSize);
+      z-index: 3;
+      margin-left: 20px;
```

Comment:
This won't do anything as the pseudo element is absolutely positioned; can probs just remove it